spec/commands/get_review_recommendations_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+describe GetReviewRecommendations do
+  def perform(*args)
+    described_class.for(*args)
+  end
+
+  def perform_with_predefined_args
+    perform(
+      github_user_id: github_user_id,
+      other_users_ids: other_users_ids
+    )
+  end
+
+  let(:github_user_id) { 1 }
+  let(:other_users_ids) { [2, 3, 4, 5, 6, 7] }
+
+  before do
+    allow_any_instance_of(GetReviewRecommendations)
+      .to receive(:times_reviewed_by_each_user).and_return([1, 4, 6, 3, 5, 1])
+    allow_any_instance_of(GetReviewRecommendations)
+      .to receive(:best_recommendations).and_return([3, 4, 6])
+    allow_any_instance_of(GetReviewRecommendations)
+      .to receive(:worst_recommendations).and_return([2, 5, 7])
+  end
+
+  context 'perform get review recommendation' do
```

Comment:
en general los context se usan para probar ciertas condiciones de la prueba. En este caso me imagino por ejemplo

- context 'with other_users_id empty'
- context 'with other_users_id not empty'
  - context 'user without reviews'
  - context 'user with reviews'

etc.

La idea de los `context` es ir definiendo tests para las condiciones posibles que pueden afectar el resultado de la función. Hay que ir definiendo el "árbol" de posibles condiciones con los context.